src/v2/Apps/Search/Server/redirectQueryToTerm.ts

```diff
@@ -1,19 +1,15 @@
 import { stringify } from "qs"
 
-export const searchMiddleware = async (req, res, next) => {
-  if (!req.query.term) {
+export function redirectQueryToTerm({ req, res, next }) {
```

Comment:
maybe better to specify the argument type and destructure only the variables that the fn is going to use, we can remove next from here in this case